www/FabricPlugin.Crashlytics.js

```diff
@@ -21,8 +21,48 @@ var FabricCrashlytics = (function () {
         }
     }, {
         key: 'sendNonFatalCrash',
-        value: function sendNonFatalCrash(message) {
-            window.fabric.core.execPlugin('sendNonFatalCrash', [message]);
+        value: function sendNonFatalCrash(message, trace) {
+            var params = [message];
+
+            if (trace) {
+                // validate trace (easier here)
+                var tmp = [];
+
+                var _iteratorNormalCompletion = true;
+                var _didIteratorError = false;
+                var _iteratorError = undefined;
+
+                try {
+                    for (var _iterator = trace[Symbol.iterator](), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
```

Comment:
Hi there,

Can you explain the use of an iterator here instead of a simple for loop ?